packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -23,6 +23,8 @@ export const messages = {
     'To run this command, first install the @salesforce/lwc-dev-server plugin. For more info, see [https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev](https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev).',
   force_lightning_lwc_start_addr_in_use:
     "The local development server couldn't start as the address is already in use. To resolve, do one of these\n 1) Stop the local dev server running on any another instance.\n or 2) Change the default port [Configuration for Projects (Optional)](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.get_started_local_dev_setup).\n 3) Kill the process running on the specified port.",
+  force_lightning_lwc_inactive_scratch_org:
+    'The local development server couldn’t start as your scratch org is not active. Run "SFDX: Create a Default Scratch Org" to create a scratch org, or run "SFDX: Set a Default Org" to switch to an active scratch org.',
```

Comment:
@ekapner I updated the user messaging here. Would you have additional feedback?

Reply:
I worry about providing a URL to docs that could become obsolete at some future point in time. So, let me see if we can be more generic.

Reply:
@ekapner it's a valid concern but this is better than not giving users a path forward and expecting them to search the internet for an answer. Most of the urls we provide are also under our control which helps in mitigating the problems. When we've ran into the situation of urls being out of date we're able to address it pretty fast.

Reply:
Are you referring to the URL in `force_lightning_lwc_start_addr_in_use`?

Reply:
'To run this command, install the @salesforce/lwc-dev-server plugin. For more info, see the Local Development Server topics in the Lightning Web Components documentation.,
  force_lightning_lwc_start_addr_in_use:
    "The local development server can't start because the address is already in use. To fix, try one of these options: \n 1) Stop the local dev server running on any another instance.\n or 2) Change the default port see the Local Development Server topics in the Lightning Web Components documentation.\n 3) Kill the process running on the specified port.",
  force_lightning_lwc_inactive_scratch_org:
    'The local development server can't start because your scratch org is not active. Run "SFDX: Create a Default Scratch Org" to create a scratch org, or run "SFDX: Set a Default Org" to select an active scratch org.',

Reply:
@lcampos @xyc - ok, our style standards are to avoid the direct URLs - but if you both feel like there's low risk here, let's keep them for now. I agree, it's always desirable to point users directly to the info they need.

Reply:
Thanks @ekapner . I'll incorporate your comments & keep the URLs. In general I agree with @lcampos that providing a url is more actionable for users. I also wonder if it would be possible to provide a link redirect when such deprecation happens.